content/docs/user-guide/action-manager/fundamentals/architecture/visibility.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Action Visibility
+linktitle: Visibility
+description: 
+weight: 204
+---
+
+The visibility properties of an action determines when actions are displayed in the User Interface as part of a toolbar or a menu based on their active state and assigned Action Context Modes.
+These behaviors are dynamic, as in they will be re-evaluated whenever the state of the Editor changes and an action's enabled state is toggled, or the currently active Action Context Mode changes.
+
+
+## Visibility Setting Values
+
+The Visibility properties of an action can be set to one of three values.
+
+
+#### AlwaysShow
+
+The Action will always be shown as part of the menus or toolbars it is assigned to.
+In case the action's enabled state is set to false, or the action is assigned to action context modes different from the currently active one, the action will be visible but greyed out/inaccessible.
```

Comment:
```suggestion
The action always shows as part of the menus or toolbars it is assigned to.
In case the action's enabled state is set to false, or the action is assigned to action context modes different from the currently active one, the action will be visible but greyed out and inaccessible.
```